RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerItemViewModel.cs

```diff
@@ -32,17 +32,17 @@ public class CompareByType : Comparer<CodeExplorerItemViewModel>
         private static readonly Dictionary<DeclarationType, int> SortOrder = new Dictionary<DeclarationType, int>
         {
             {DeclarationType.LibraryFunction, 0},
-            {DeclarationType.LibraryProcedure, 1},
-            {DeclarationType.UserDefinedType, 2},
-            {DeclarationType.Enumeration, 3},
-            {DeclarationType.Event, 4},
-            {DeclarationType.Constant, 5},
-            {DeclarationType.Variable, 6},
-            {DeclarationType.PropertyGet, 7},
-            {DeclarationType.PropertyLet, 8},
-            {DeclarationType.PropertySet, 9},
-            {DeclarationType.Function, 10},
-            {DeclarationType.Procedure, 11}
+            {DeclarationType.LibraryProcedure, 0},
+            {DeclarationType.UserDefinedType, 1},
+            {DeclarationType.Enumeration, 2},
+            {DeclarationType.Event, 3},
+            {DeclarationType.Constant, 4},
+            {DeclarationType.Variable, 5},
+            {DeclarationType.PropertyGet, 6},
+            {DeclarationType.PropertyLet, 7},
+            {DeclarationType.PropertySet, 8},
+            {DeclarationType.Function, 9},
+            {DeclarationType.Procedure, 9}
         };
```

Comment:
The values were unique in the original source, but when it comes to sorting in the tree, we want a `Function` and a `Sub` to be grouped together.

Is this not a valid way of achieving that objective?

Reply:
I kind of dislike it, since the order isn't absolute anymore. Not that it matters really much. In either case a comment should be added to explain the "weirdness"

Reply:
Comment added to the top of the list